pkg/gather/clusterconfig/sap_config.go

```diff
@@ -53,8 +50,7 @@ func gatherSAPConfig(ctx context.Context, dynamicClient dynamic.Interface, coreC
 	sccToGather := []string{"anyuid", "privileged"}
 	crbToGather := []string{"system:openshift:scc:anyuid", "system:openshift:scc:privileged"}
 
-	datahubsList, err := dynamicClient.Resource(datahubGVR).List(ctx, metav1.ListOptions{})
-
+	datahubsList, err := dynamicClient.Resource(datahubGroupVersionResource).List(ctx, metav1.ListOptions{})
```

Comment:
Can you please do the similar change also for `pkg/gather/clusterconfig/sap_pods.go`? Thanks

Reply:
done